Lab1/README.md

```diff
@@ -114,6 +114,10 @@ But there are many other places that will let you run jupyter notebooks on their
 
 - [ ] **Deliverable:** Demonstrate somehow that you've used each of these notebook services.
 
+## AWS SAGEMAKER TUTORIAL
+
+Below is the link for the super awesome AWS SageMaker Tutorial:
+[Here is the tutorial!](https://aws.amazon.com/getting-started/tutorials/build-train-deploy-machine-learning-model-sagemaker/?sc_icampaign=pac-sagemaker-console-tutorial&sc_ichannel=ha&sc_icontent=awssm-2276&sc_iplace=console-body&trk=ha_awssm-2276)
```

Comment:
Please removing the tracking garbage from the url

Reply:
i.e., simplify to this: https://aws.amazon.com/getting-started/tutorials/build-train-deploy-machine-learning-model-sagemaker/